src/media/css/pageheader.styl

```diff
@@ -0,0 +1,34 @@
+.pageheader
+  align-items flex-end
+  border-bottom 1px solid #DDD
+  display flex
+  justify-content space-between
+  
+  h1
+    font-size 28px
+    font-weight 400
+    margin 0
+
```

Comment:
maybe we should start double line-breaking between groups? it's starting to get hard to parse since a single-line break doesn't have as much meaning in terms of logical separation